### PR TITLE
configurable redirect page

### PR DESCRIPTION
**Pull request by jessopb**

### CODE REVIEW

File: web/redirect.html [hunk 1]
```diff
@@ -16,43 +14,181 @@
...
+        </div>
+      </div>
+      <div class="controlelement">
+        <div class="checkbox-element">
+          <input id="remember" name="remember" type="checkbox">
+          <label for="remember">Always do this</label>
+          <div class="checkbox-toggle"></div>
+        </div>
+      </div>
+      <p>If you do not have the LBRY app, you can <a class="button--primary" href="https://lbry.io/get" rel="noopener noreferrer" target="_blank">download it here</a></p>
```
Comment (kauffj) [inline web/redirect.html]: Please replace all instances of lbry.io with lbry.com

File: web/redirect.html [hunk 2]
```diff
@@ -16,43 +14,181 @@
...
+
+      // event listeners
+      appButton.addEventListener("click", handleChoice);
+      webButton.addEventListener("click", handleChoice);
+      stopButton.addEventListener("click", stop);
+      rememberBox.addEventListener("click", handleRemember);
+
+      dest.innerHTML = "LBRY.tv";
+
+      if (storedDest) {
```
Comment (kauffj) [inline web/redirect.html:123]: what if `storedDest` is neither `appButton` nor `webButton`?

File: web/redirect.html [hunk 3]
```diff
@@ -16,43 +14,181 @@
...
+      let go = true;
+      let storedDest = localStorage.getItem("lbrytarget"); //webButton or appButton
+
+      let choice = storedDest || "webButton";
+      let prefix = appPrefix;
+
+      // elements
+      const dest = document.getElementById('dest');
+      const timer = document.getElementById('timer');
+      const appButton = document.getElementById("appButton");
```
Comment (kauffj) [inline web/redirect.html:107]: (I don't care that much and I can understand if this project isn't worth linting, but there's inconsistent quotation marks here and in a variety of other places)

File: web/redirect.html [hunk 4]
```diff
@@ -16,43 +14,181 @@
...
+          prefix = tvPrefix;
+        } else {
+          prefix = appPrefix;
+        }
+      }
+
+      function handleRemember() {
+        stop();
+        !rememberBox.checked && localStorage.removeItem('lbrytarget');
+        !rememberBox.checked && console.log('clearing preference');
```
Comment (kauffj) [inline web/redirect.html]: is it just cool to leave `console.log` in production now? am I a dinosaur?

File: web/redirect.html [hunk 5]
```diff
@@ -16,43 +14,181 @@
...
+      }
+
+      function handleChoice(event) {
+        stop();
+        choice = event.target.id;
+        dest.innerHTML = (choice === "appButton") ? "your LBRY App" : "LBRY.tv";
```
Comment (kauffj) [inline web/redirect.html]: "appButton" and "webButton" should be consts if we're [abiding the maxims](https://gist.github.com/lyoshenka/0a43205aa9a072b196ff87e2c689a8b9)

**Review by kauffj — CHANGES_REQUESTED** [review]
The code is pretty good, but I think there are some UI/UX improvements that are worth making. Will discuss in chat.

File: web/redirect.html [hunk 6]
```diff
@@ -4,55 +4,183 @@
...
   </head>
 
   <body>
-    <section>
-      <h1>Buckle up, gang!</h1>
-      <p>Redirect in progress...</p>
-      <p>If you do not have the LBRY app, you can <a href="https://lbry.io/get" rel="noopener noreferrer" target="_blank">download it here</a>.</p>
+    <section class="container">
+      <header class="container__header">
+        <button class="button--primary" id="stopButton">
```
Comment (kauffj) [inline web/redirect.html]: I think making this a button is a mistake. This results in 4 primary styled buttons inside of the same component. There should probably only be 1 or 2.

File: web/redirect.html [hunk 7]
```diff
@@ -4,55 +4,183 @@
...
+        <button class="button--primary" id="webButton">LBRY.tv (beta)</button>
+      </div>
+
+      <checkbox-element>
+        <input id="remember" name="remember" type="checkbox">
+        <label for="remember">Always do this</label>
+        <checkbox-toggle/>
+      </checkbox-element>
+
+      <p>Don't have the LBRY app? <a class="button--primary" href="https://lbry.com/get" rel="noopener noreferrer" target="_blank">Download it here</a></p>
```
Comment (kauffj) [inline web/redirect.html]: Please make this a text link.

File: web/redirect.html [hunk 8]
```diff
@@ -4,55 +4,183 @@
...
-      <h1>Buckle up, gang!</h1>
-      <p>Redirect in progress...</p>
-      <p>If you do not have the LBRY app, you can <a href="https://lbry.io/get" rel="noopener noreferrer" target="_blank">download it here</a>.</p>
+    <section class="container">
+      <header class="container__header">
+        <button class="button--primary" id="stopButton">
+          Sending you to <span id="dest"></span> in <span id="timer"></span>
+        </button>
+
+        <small>Click button to stop timer</small>
```
Comment (kauffj) [inline web/redirect.html]: A button or link that says "Stop" would not need this reminder text

File: web/redirect.html (hunk 7 above)
Comment (kauffj) [reply web/redirect.html]: And can this be `meta` or `help` or something smaller than typical `<p>`?

**Review by kauffj — CHANGES_REQUESTED** [review]
@NetOperatorWibby can you do one more set of small changes on this?

File: web/redirect.html (hunk 8 above)
Comment (NetOpWibby) [reply web/redirect.html]: Why is this text bad? What would a better interface look like?

Comment (kauffj) [reply web/redirect.html]: If there's a button and/or link that says stop, then you do not need hint text to tell me what to do.

**Pull request merged**